src/test/java/com/edmunds/tools/databricks/maven/UpsertJobMojoTest.java

```diff
@@ -202,14 +180,15 @@ public void testGetJobId_none() throws Exception {
         assertThat(jobId, nullValue());
     }
 
-    @Test
+    // TODO rewrite
```

Comment:
I think rewriting these tests should be required for this merge

Reply:
Done. But now these tests check SettingsUtils methods.
Although JobMojo initialization still crucial for them